ultimatemedia.js.org

C1200 opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```ultimatemedia.js.org``` to make it work.
I'll merge after we have "contact".

C1200 commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `ultimatemedia.js.org` to make it work.
> I'll merge after we have "contact".

Hi!
Thanks for replying! I have added a cname file to my repository as you requested.

indus commented:
Welcome https://ultimatemedia.js.org
